api/v1/paths/developers/{developerId}.js

```diff
@@ -40,6 +40,25 @@ const del = async (req, res) => {
   }
 };
 
+/**
+ * @summary Patch developer by unique id
+ */
+const patch = async (req, res) => {
+  try {
+    const { developerId } = req.params;
+    const result = await developersDao.patchDeveloper(developerId, req.body);
+    if (result.rowsAffected < 1) {
+      errorBuilder(res, 404, 'A developer with the specified ID was not found.');
+    } else {
+      const updatedResult = await developersDao.getDeveloperById(developerId);
+      res.send(updatedResult);
+    }
+  } catch (err) {
+    errorHandler(res, err);
+  }
+};
+
 get.apiDoc = paths['/developers/{developerId}'].get;
+patch.apiDoc = paths['/developers/{developerId}'].patch;
```

Comment:
Do we also need to add the apiDoc to `del`?

Reply:
I must have missed that. Added it now.